src/plugins/plot/configuration/XAxisModel.js

```diff
@@ -20,22 +20,26 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 import Model from "./Model";
+
 /**
-     * TODO: doc strings.
-     */
+ * @extends {Model<XAxisModelType>}
+ */
 export default class XAxisModel extends Model {
+    /**
+     * @param {XAxisModelOptions} options
+     */
     initialize(options) {
         this.plot = options.plot;
         this.set('label', options.model.name || '');
-        this.on('change:range', function (newValue, oldValue, model) {
-            if (!model.get('frozen')) {
-                model.set('displayRange', newValue);
+        this.on('change:range', (newValue) => {
+            if (!this.get('frozen')) {
```

Comment:
Actually, let's make these class member methods instead.

Reply:
Moved your comment to https://github.com/nasa/openmct/pull/4789/files#r792995330